Sources/Showcase/PlaygroundListView.swift

```diff
@@ -377,16 +382,17 @@ public struct PlaygroundNavigationView: View {
     }
 
     public var body: some View {
-        NavigationStack {
-            List(matchingPlaygroundTypes(), id: \.self) { playground in
-                NavigationLink(value: playground, label: { Text(playground.title) })
-            }
-            .navigationTitle(Text("Showcase"))
-            .navigationDestination(for: PlaygroundType.self) {
-                $0.navigationTitle(Text($0.title))
-            }
-            .searchable(text: $searchText)
-        }
+        CombineLatestPlayground()
+//        NavigationStack {
```

Comment:
I think this code needs to be restored.

Reply:
Sorry I should have commented, because you're totally right. This isn't meant to be merged in its current state. I had to comment out the rest of the playgrounds because the example wouldn't work because it uses a nested `NavigationStack` that would conflict with this one in `PlaygroundListView`.

Reply:
I got this working within a few hours with AsyncStream so problem solved! Closing this.